index.html

```diff
@@ -5,8 +5,48 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     
+<header>
+    <nav>
+        <div class="name">
+            <div class="name-text">Sungabanja Thawethe</div>
+        </div>
+        <div class="item-navigation"> 
+            <div class="frame-port">
+                <div class="portfolio">Portfolio</div>
+            </div>
+            <div class="frame-about">
+                <div class="about">About</div>
+            </div>
+            <div class="frame-contact">
+                <div class="contact">Contact</div>
+            </div>       
+        </div>
+        <div class="email-box">
+            <img class="icons" src="Union.png" alt="Github icon">
+            
+        </div>
+    </nav>
+</header>
+
+<main>
+    <aside>
+        <img class="icons" src="github.png" alt="Github icon">
+        <img class="icons" src="Linkedin icon.png" alt="Github icon">
+        <img class="icons" src="angel list.png" alt="Github icon">
+        <img class="icons" src="twitter.png" alt="Github icon">
+        <img class="icons" src="medium.png" alt="Github icon">
+    </aside>
+
+    <div class="main-text">
+        <div class="hello-me">Hey There. I'm Sunga</div>
+        <div class="SD">I'm a Software Developer</div>
+        <div class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
```

Comment:
- I think it will be better to use `<hx>` and `<p>` tags to represent text instead of divs alone. Why don't we replace the divs to `<h1>` and `<p>` in the cases that apply?  

For example, line 47 should be a paragraph and in a `<p>` tag. 🤗